doc/ref_cert/RC1/chapters/chapter02.rst

```diff
@@ -567,7 +565,7 @@ different types of requirements and system properties:
    validating the functional correctness of the system under test. API
    compliance test cases exercise only the specific well-defined APIs
    described in the reference architecture (see `Interfaces and
-   APIs <https://cntt.readthedocs.io/en/latest/ref_arch/openstack/chapters/chapter05.html>`__).
+   APIs :doc:`ref_arch/openstack/chapters/chapter05`).
```

Comment:
Row 567, the open quote for Interfaces needs to be taken care of in row 568

Reply:
@pgoyal01 The exact title will be written automatically. I can send you an archive including all html files if you will.

Reply:
@collivier Line 567 states, "described in the reference architecture ... " - [lease see the char before Interfaces and the ` needs to be closed in the next line.